connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -440,6 +441,26 @@ message NAFill {
   repeated Expression.Literal values = 3;
 }
 
+
+// Drop rows containing null values.
+// It will invoke 'Dataset.na.drop' (same as 'DataFrameNaFunctions.drop') to compute the results.
+message NADrop {
```

Comment:
since parameter `how: String` can be expressed by `min_non_nulls` https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/DataFrameNaFunctions.scala#L491-L497 ,

only `min_non_nulls` is added in proto message

Reply:
In this case, can you document on top  the that `optional int32 min_non_nulls = 4;` for clients. E.g. string `any` = 1.

Reply:
nice, will update this doc